src/measure_theory/tactic.lean

```diff
@@ -124,11 +124,13 @@ do t ← tactic.target,
   | _ := skip
   end
 
-/-- List of tactics used by `measurability` internally. -/
+/-- List of tactics used by `measurability` internally. The option `use_exfalso := ff` is passed to
+the tactic `apply_assumption` in order to avoid loops in the presence of negated hypotheses in
+the context. -/
 meta def measurability_tactics (md : transparency := semireducible) : list (tactic string) :=
 [
-  propositional_goal >> apply_assumption
-                        >> pure "apply_assumption",
+  propositional_goal >> tactic.interactive.apply_assumption none {use_exfalso := ff}
+                        >> pure "apply_assumption none {use_exfalso := ff}",
```

Comment:
```suggestion
                        >> pure "apply_assumption {use_exfalso := ff}",
```
This `none` shouldn't be necessary -- the way `apply_assumption` is defined is wrong. I'm fixing that and will rebase this PR on top of that change!